### DISCUSSION TITLE
GitHub Education benefits

### DISCUSSION THREAD

**Discussion by rootcode-creator** [community/community#140176] [GitHub Education]
### Hi everyone,

Hi status of my student Developer Pack Application shows  **Congrats! Your request for GitHub Education benefits was approved and your benefits will be available soon** but till today it didn't  add to my account

**Comment by Wes5000** [comment]
> ### Hi everyone,
> 
> Hi status of my student Developer Pack Application shows  **Congrats! Your request for GitHub Education benefits was approved and your benefits will be available soon** but till today it didn't  add to my account

Awesome 👍

**Comment by davevad93** [comment]
Hi @rootcode-creator ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @rootcode-creator was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
